code/modules/mob/living/simple_animal/guardian/guardian.dm

```diff
@@ -72,18 +72,18 @@
 			loc = get_turf(summoner)
 
 
-/mob/living/simple_animal/hostile/guardian/adjustBruteLoss(amount) //The spirit is invincible, but passes on damage to the summoner
+/mob/living/simple_animal/hostile/guardian/adjustCloneLoss(amount) //The spirit is invincible, but passes on damage to the summoner
```

Comment:
I think you meant to keep this as adjustBruteLoss(). We want conventional damage, not clone damage, dealt to the guardian to give clone damage to the owner.